### PR TITLE
release-v1.0: update redirect and release menu

### PR DESCRIPTION
**Pull request by nojnhuh**
frontport(?) of #234 since the site still seems to redirect to v0.11

### CODE REVIEW

**Review by snehachhabria — APPROVED** [review]

**Comment by nojnhuh** [comment]
Need to wait until https://release-v1-0.docs.openservicemesh.io/ is up before merging this

**Pull request merged**
